reproman/resource/singularity.py

```diff
@@ -65,6 +65,16 @@ def connect(self):
             self.id = None
             self.status = None
 
+    def _run_instance_command(self, cmd, args=[], **kwargs):
+        """Singularity 3.x changed from "instance.cmd" to just "instance cmd"
+        This is a helper to centralize execution
+        """
+        cmd = ['instance.%s' % cmd] \
+            if external_versions['cmd:singularity'] < '3' \
+            else ['instance', cmd]
```

Comment:
yeah, I am somewhat on the fence every such time ... greedy me nearly always wins to avoid duplication.  But indeed it might be better to switch in general to more conventional one unless `X if ... else ` is very short/obvious